src/bioregistry/data/bioregistry.json

```diff
@@ -21852,6 +21852,110 @@
     "no_own_terms": true,
     "repository": "https://github.com/MIT-LCP/mimic-omop"
   },
+  "cohesindb.binding": {
+    "contact": {
+      "email": "rnakato@iqb.u-tokyo.ac.jp",
+      "name": "Ryuichiro Nakato",
+      "orcid": "0000-0003-3019-5817"
+    },
+    "contributor": {
+      "email": "maaya.m466@gmail.com",
+      "github": "maayamanoj466",
+      "name": "Maaya Manoj",
+      "orcid": "0009-0000-6974-6386"
+    },
+    "description": "CohesinDB is a multiomics cohesin database in human cells. CohesinDB includes 2043 epigenomics, transcriptomics and 3D genomics datasets from 530 studies involving 176 cell types. Each identifier represents a single cohesin binding cite. Each cohesin object is annotated with locus, cell type, classification, function, 3D genomics and cis-regulatory information.",
```

Comment:
Please rephrase the description to first say what kind of entities are in the semantic space. Then after you can describe the database that the semantic space is from

A guide towards writing a good description can be found here: https://github.com/biopragmatics/bioregistry/blob/main/docs/CONTRIBUTING.md#writing-a-good-description